pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -307,9 +305,17 @@ func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems
 			pod.Name, item.ClusterName, temp.Phase)
 	}
 
-	if runningFlag {
+	// More details please refer to: https://github.com/karmada-io/karmada/issues/2137
```

Comment:
```suggestion
	// Check final phase in order: PodFailed-->PodPending-->PodRunning-->PodSucceeded
	// Ignore to check PodUnknown as it has been deprecated since year 2015.
	// More details please refer to: https://github.com/karmada-io/karmada/issues/2137
```
We should give a basic description before leading people to more details.